src/components/playground/FontsList.js

```diff
@@ -32,8 +32,8 @@ function FontsList(props) {
       if (!hasFont) return false;
 
       setFonts((prevFonts) => ({
-        ...prevFonts,
         ...{ [formattedName]: [] },
+        ...prevFonts,
```

Comment:
Is there a reason you swapped this line?

Reply:
Yes, I thought you want new fonts added by user to be on the top of the list... prevFonts is array of old "default" fonts

Reply:
Can u give me please more information what are default fonts in this case? Are you using google fonts api for fonts?

Reply:
Yes we are using the google fonts api to load the `custom fonts`. I just checked your PR and selecting the Roboto font works fine, are you getting any errors in the console about this?

`default fonts` are the font families that are standard supported by every modern browser. Arial, Helvetica, etc.